src/azure-cli-core/azure/cli/core/tests/test_profile.py

```diff
@@ -63,7 +61,6 @@ def __init__(self, *args, **kwargs):
         self.object_id = kwargs.get('object_id')
         self.msi_res_id = kwargs.get('msi_res_id')
         self.resource = kwargs.get('resource')
-        MSRestAzureAuthStub.return_value = self
```

Comment:
Don't save the instance as class attribute so that tests won't interfere with each other.